cms/server/admin/handlers/contestuser.py

```diff
@@ -239,10 +239,17 @@ def post(self, contest_id, user_id):
 
             # Update the team
             self.get_string(attrs, "team")
-            team: Team | None = (
-                self.sql_session.query(Team).filter(Team.code == attrs["team"]).first()
-            )
-            participation.team = team
+            team_code = attrs.get("team", "")
```

Comment:
The `get_string` call above seems to have an `empty=""` parameter, does that mean that we already default to an empty string there? https://github.com/cms-dev/cms/blob/f277dcc945589a3a86c99b60c1df26fa1f3cd84d/cms/server/admin/handlers/base.py#L375C5-L375C56

If yes, then we probably don't need the `.get()` here and we can do `attrs["team"]` like before